crates/dojo-indexer/src/graphql/entity.rs

```diff
@@ -1,14 +1,22 @@
+use chrono::{DateTime, Utc};
 use juniper::{graphql_object, FieldResult};
+use serde::Deserialize;
 
 use super::entity_state::EntityState;
 use super::entity_state_update::EntityStateUpdate;
 use super::server::Context;
 use super::{entity_state, entity_state_update};
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct Entity {
     pub id: String,
     pub name: Option<String>,
+    pub partition_id: String,
+    pub partition_keys: String,
```

Comment:
Should this be:
```suggestion
    pub keys: Vec<String>,
```

Perhaps String should be a `FieldElement` too?

Reply:
Yeah will update name but looks like sqlite doesn't support string arrays, I guess the indexer can just insert csv string.

I think we'll have to define custom scalar for graphql to use FieldElement? I can add that as part of another PR.